<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiCoilItem</class>
 <widget class="QWidget" name="MultiCoilItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>35</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radioButton_ON">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>6</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_OFF">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>6</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_SerialNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>序号</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_CoilAddr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>线圈地址</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
